src/pages/Build.css

```
@@ -0,0 +1,598 @@
+  transition: transform 0.2s;
+}
+.b-expand-btn.open .b-expand-arrow { transform: rotate(180deg); }
+
+.b-prompt-full {
+  display: none;
+  padding: 0 24px 20px;
+  font-family: ui-monospace, monospace;
+  font-size: 13px;
+  line-height: 1.8;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)